src/tic_tac_toe/game.clj

```diff
@@ -4,13 +4,24 @@
             [tic-tac-toe.referee :as referee]
             [tic-tac-toe.user-interface :as ui]))
 
+(defn share-information
+  [player game-ui board]
+  (->> {:game-ui game-ui :board board}
+       ((:needed player))))
+
 (defn request-move
-  [game-ui player1]
-  (-> (player/pick-move player1 game-ui)
-      (referee/validate-move)))
+  [game-ui player board]
+  (let [game-info (share-information player game-ui board)]
+    (->> (player/pick-move player game-info)
+         (referee/validate-move player))))
+
+(defn swap-players
+  [players]
+  (assoc players :current (players :other) :other (players :current)))
```

Comment:
I am not sure if you are using `:other` elsewhere, but this could simplify things potentially. https://clojuredocs.org/clojure.core/cycle 🤔

Reply:
Experimented with cycle.  Good to know it's out there.  Ended up using [another approach](https://github.com/saralein/tic-tac-toe_clojure/pull/4/files#diff-57905e40aaa721e61a7ea7ef112d9ed3R18) though: alternating order passed to play in the recursion.